thinc/layers/add.py

```diff
@@ -4,19 +4,23 @@
 from ..config import registry
 from ..types import Array
 from ..util import get_width
+from thinc.types import Reduced_OutT
 
 
 InT = TypeVar("InT", bound=Array)
 
 
 @registry.layers("add.v0")
-def add(*layers: Model) -> Model[InT, InT]:
+def registry_add(*layer: Model) -> Model:
+    return add(*layer)
```

Comment:
If we add the decorator directly on top of `add()`, mypy can't infer the type. Not sure why, so this double function that calls the other is needed.

Reply:
That's not needed anymore after the module rename. So, back to `add()` with decorator on top.